test/buy-tshirt.spec.ts

```diff
@@ -3,30 +3,52 @@ import { MenuContentPage, ProductListPage, ProductAddedModalPage,
   SummaryStepPage, SignInStepPage, AddressStepPage,
   ShippingStepPage, OrderSummaryPage, BankPaymentPage, PaymentStepPage} from '../src/page';
 
-describe('Buy a t-shirt', () => {
-  const menuContentPage: MenuContentPage = new MenuContentPage();
-  const productListPage: ProductListPage = new ProductListPage();
-  const productAddedModalPage: ProductAddedModalPage = new ProductAddedModalPage();
-  const summaryStepPage: SummaryStepPage = new SummaryStepPage();
-  const signInStepPage: SignInStepPage = new SignInStepPage();
-  const addressStepPage: AddressStepPage = new AddressStepPage();
-  const shippingStepPage: ShippingStepPage = new ShippingStepPage();
-  const orderSummaryPage: OrderSummaryPage = new OrderSummaryPage();
-  const bankPaymentPage: BankPaymentPage = new BankPaymentPage();
-  const paymentStepPage: PaymentStepPage = new PaymentStepPage();
-  it('then should be bought a t-shirt', async () => {
+describe('This must open the shopping website in the browser', () => {
```

Comment:
It should follow a Given.. When... Then... structure...Please improve your description